docs/entities/clustered/light_type_reference.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Light Type Reference"
+features:
+    - USE_CLUSTERED
+---
+# Light Type Reference
+
+This is a reference page for the 4 light types.
+
+## Static
+
+This light type is mainly used for lights that cast baked direct and indirect light, with no need for dynamic shadows or any specular lighting. The Shadowed spawnflag is non-functional with this light type.
```

Comment:
I suggest adding a note about the lights being the same as standard source engine lights.

```suggestion
This light type is mainly used for lights that cast baked direct and indirect light, with no need for dynamic shadows nor any specular lighting. The Shadowed spawnflag is non-functional with this light type.

These lights are the same as standard Source Engine lights.
```